spec/models/cart_item_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe CartItem, type: :model do
+  let(:cart_item) {create(:cart_item, pricing: pricing, quantity: quantity)}
+  let(:pricing) {}
+  let(:quantity){10}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.